shiny/ui/_page.py

```diff
@@ -443,6 +444,99 @@ def page_bootstrap(
     )
 
 
+def page_auto(
+    *args: TagChild | TagAttrs,
+    _page_fn: Callable[..., Tag] | None = None,
+    _fillable: bool = False,
+    _full_width: bool = False,
+    **kwargs: object,
```

Comment:
Or don't take `_page_fn`

Reply:
I considered taking out `page_fn`, but I think we should keep it because it's useful when, in Express, you call `page_opts(page_fn=...)`. The `page_opts` function can just pass it through to this function.